packages/cli/src/UserManagement/routes/index.ts

```diff
@@ -53,7 +53,7 @@ export function addRoutes(this: N8nApp, ignoredEndpoints: string[], restEndpoint
 			req.url.startsWith('/fonts/') ||
 			req.url.startsWith(`/${restEndpoint}/settings`) ||
 			req.url.startsWith(`/${restEndpoint}/resolve-signup-token`) ||
-			req.url === `/${restEndpoint}/user`
+			(req.method === 'POST' && new RegExp(`/${restEndpoint}/users/[\\w\\d-]*`).test(req.url))
```

Comment:
I think this whole part needs to be deleted here, because this middleware is about authentication and not what endpoints are just accessible by the owner

Reply:
Sorry, not sure I understand what you mean here.

Removing this line triggers 401 for all `POST /users/:id` tests as I'd expect.

- `POST /users/:id` (originally `POST /user`) is authless
- `POST /users` has auth
- `POST /users/:id/reinvite` has auth

Auth middleware should skip only the first case.

Reply:
Apologies for the glitch that removed your comment. You were asking to remove this line.

Afaik `POST /users/:id` (originally `POST /user`) is authless, so this exception

```ts
(req.method === 'POST' && new RegExp(`/${restEndpoint}/users/[\\w\\d-]*`).test(req.url))
``` 

is needed to prevent Passport from returning 401 for `POST /users/:id`. Removing this exception restores protection for the endpoint, causing all three tests to break as expected.

Did you mean something else?

Reply:
my mistake, checked in the auth contract and the endpoint needs to be authless as you are having it 👍🏼